corehq/apps/users/forms.py

```diff
@@ -781,11 +781,15 @@ def __init__(self, css_id, source_css_id, attrs=None):
         super(PrimaryLocationWidget, self).__init__(attrs)
         self.css_id = css_id
         self.source_css_id = source_css_id
+        self.submit_form_id = submit_form_id
+        self.empty_primary_location_message = empty_primary_location_message
```

Comment:
Nit: Does this need to be parameterized?  What's the advantage over just defining it right here?